homeassistant/components/tuya/alarm_control_panel.py

```diff
@@ -89,51 +86,65 @@ def async_discover_device(device_ids: list[str]) -> None:
 class TuyaAlarmEntity(TuyaEntity, AlarmControlPanelEntity):
     """Tuya Alarm Entity."""
 
-    entity_description: TuyaAlarmControlPanelEntityDescription
-    _attr_supported_features = (
-        SUPPORT_ALARM_ARM_HOME | SUPPORT_ALARM_ARM_AWAY | SUPPORT_ALARM_TRIGGER
-    )
+    entity_description: AlarmControlPanelEntityDescription
+    _attr_icon = "mdi:security"
+    _attr_supported_features = 0
+
+    state_mapping = {
+        Mode.ALARM_DISARMED: STATE_ALARM_DISARMED,
+        Mode.ALARM_ARM: STATE_ALARM_ARMED_AWAY,
+        Mode.ALARM_HOME: STATE_ALARM_ARMED_HOME,
+        Mode.ALARM_SOS: STATE_ALARM_TRIGGERED,
+    }
```

Comment:
This is a static mapping, which fit better as a constant (instead of having it in memory for each instance of this entity separately).